docs/notes/05-linters-and-tests.md

```diff
@@ -23,6 +23,22 @@ We require the following linters to be run on the code and included as checks in
 - [isort](https://pycqa.github.io/isort/)
 - [pydocstyle](http://www.pydocstyle.org/en/latest/)
 
+!!! warning "Beware of Auto-generated Scripts"
+    Auto-generated scripts may not adhere to linting standards, leading to failures. For instance, the `version.py` script generated by `setuptools-scm` ([packaging note](02-packaging.md)) doesn't comply with `black` linting standards and doesn't follow the numpy conventions, resulting in `black` and `pydocstyle` failures.
+
+    To circumvent this, you can exclude the problematic file:
+
+    - **Exclude problematic files, e.g., `version.py`**: Update the `pyproject.toml` configuration to skip such files during linting:
+      ```toml
+      [tools.black]
+      # other configurations...
+      exclude =  "/version.py$"
```

Comment:
Why the slash here?

Reply:
this is to exclude just "version.py" and not any file that ends with "version.py". 
"/version.py" matches any "path/version.py" but not "path/somehting_version.py".
I checked it with a windows OS with a github action and works on windows too